timeseries/tests/smoketests/test_all_models.py

```diff
@@ -55,20 +54,24 @@ def generate_train_and_test_data(
     return train_data, test_data
 
 
-DUMMY_MODEL_HPARAMS = {"max_epochs": 1, "num_batches_per_epoch": 1, "use_fallback_model": False}
+DUMMY_MODEL_HPARAMS = {
+    "max_epochs": 1,
+    "num_batches_per_epoch": 1,
+    "use_fallback_model": False,
+}
 
 ALL_MODELS = {
```

Comment:
I wonder whether we can reduce the space even further here. There are many models (e.g., all local models) that don't change their behavior when the covariates are present. For them, it's probably fine to just test "works with no covariates" and "works with all covariates". We can reserve the more expensive tests with all covariate combinations to other models.

No need to address this now, but we can add a TODO in case we want to tackle this later.